README.md

```diff
@@ -368,6 +368,7 @@ MOB_NOTIFY_COMMAND="/usr/bin/osascript -e 'display notification \"%s\"'"
 MOB_NOTIFY_MESSAGE="mob next"
 MOB_NEXT_STAY=true
 MOB_START_CREATE=false
+MOB_START_WITH_CI_SKIP=true
```

Comment:
As discussed, I don't think we need this config code.
We want the CI to be skipped everytime.

Reply:
OK, I understood but I thought it is convenient for preventing test codes from being broken.
I agree less code is better. I will fix.

Reply:
Fix: c298999